thecourseforum/theCourseForum
# Add react

# What's changed
- added react support to our project
- can embed react components in existing slim files

# How to test
1. `docker-compose up --build`
2. Navigate to the About page and verify that the UVA Parents Fund thank you message is rendering correctly (with image). This may take a while on first load because everything is getting cached for the first time.
